File: Go/uri_1074.go
package main

import "fmt"

func main() {
	var n = 0
	fmt.Scanln(&n)
	v := make([]int, n)
	for i := 0; i < len(v); i++ {
		fmt.Scanln(&v[i])
	}
	for i := 0; i < len(v); i++ {
		if v[i] == 0 {
			fmt.Printf("NULL\n")
		} else if v[i]%2 == 0 {
			if v[i] > 0 {
				fmt.Printf("EVEN POSITIVE\n")
			} else {
				fmt.Printf("EVEN NEGATIVE\n")
			}
		} else {
			if v[i] > 0 {
				fmt.Printf("ODD POSITIVE\n")
			} else {
				fmt.Printf("ODD NEGATIVE\n")
			}
		}
	}
}

File: Python/uri_1074.py
n = int(input())
v = []
for i in range(0, n):
    v.append(int(input()))
for i in range(0, len(v)):
    if v[i] == 0:
        print('NULL')
    elif v[i] % 2 == 0:
        if v[i] > 0:
            print('EVEN POSITIVE')
        else:
            print('EVEN NEGATIVE')
    else:
        if v[i] > 0:
            print('ODD POSITIVE')
        else:
            print('ODD NEGATIVE')